doc/neps/nep-0000.rst

```diff
@@ -0,0 +1,219 @@
+=======================
+NEP Purpose and Process
+=======================
+
+:Author: Jarrod Millman <millman@berkeley.edu>
+:Status: Draft
+:Type: Process
+:Created: 2017-12-11
+
+
+What is a NEP?
+--------------
+
+NEP stands for NumPy Enhancement Proposal.  A NEP is a design
+document providing information to the NumPy community, or describing
+a new feature for NumPy or its processes or environment.  The NEP
+should provide a concise technical specification of the feature and a
+rationale for the feature.
+
+We intend NEPs to be the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for
+documenting the design decisions that have gone into NumPy.  The NEP
+author is responsible for building consensus within the community and
+documenting dissenting opinions.
+
+Because the NEPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal [1]_.
+
+
+Types
+^^^^^
+
+There are three kinds of NEP:
+
+1. A **Standards Track** NEP describes a new feature or implementation
+   for NumPy.
+
+2. An **Informational** NEP describes a NumPy design issue, or
+   provides general guidelines or information to the NumPy community,
+   but does not propose a new feature.  Informational NEPs do not
+   necessarily represent a NumPy community consensus or
+   recommendation, so users and implementers are free to ignore
+   Informational NEPs or follow their advice.
+
+3. A **Process** NEP describes a process surrounding NumPy, or
+   proposes a change to (or an event in) a process.  Process NEPs are
+   like Standards Track NEPs but apply to areas other than the NumPy
+   language itself.  They may propose an implementation, but not to
+   NumPy's codebase; they often require community consensus; unlike
+   Informational NEPs, they are more than recommendations, and users
+   are typically not free to ignore them.  Examples include
+   procedures, guidelines, changes to the decision-making process, and
+   changes to the tools or environment used in NumPy development.
+   Any meta-NEP is also considered a Process NEP.
+
+
+NEP Workflow
+------------
+
+The NEP process begins with a new idea for NumPy.  It is highly
+recommended that a single NEP contain a single key proposal or new
```

Comment:
contains

Reply:
I slightly prefer it without an s.  If anyone else votes to change it, I will do so.